AdobeUi/view/adminhtml/web/js/components/grid/column/overlay.js

```diff
@@ -3,8 +3,9 @@
  * See COPYING.txt for license details.
  */
 define([
-    'Magento_Ui/js/grid/columns/column'
-], function (Column) {
+    'Magento_Ui/js/grid/columns/column',
+    'ko'
```

Comment:
I don't see knockout usages in this component, I think `ko` dependency needs to be removed